### ISSUE TITLE
AssignsNote's inspect is executing ActiveRecord::Relation query

### ISSUE THREAD

**Issue by Willianvdv** [indirect/rails-footnotes#102]
Not sure this is actually a bug, but I'm curious in your thoughts about this issue. 

In my controller I do something like this:

```
def index
   @products = Products.where('1=1') # There are a lot of products!
end
```

In my view I don't use @products. I expect that the query isn't executed, but the `AssignsNote` inspects the value exectuting the query. See:
https://github.com/josevalim/rails-footnotes/blob/master/lib/rails-footnotes/notes/assigns_note.rb#L48

Is there a way that a ActiveRecord::Relation's query doesn't gets executed?

**Comment by philipnery** [comment]
This issue is probably the culprit behind my problem in [ActiveAdmin] (https://github.com/activeadmin/activeadmin) 
activeadmin/activeadmin#3898

**Comment by philipnery** [comment]
If anyone's interested, I made a fork that will add a limit in assigns note [here] (https://github.com/philipnery/rails-footnotes). It will only add a limit if there's no limit specified. The catch here is that it will add additional queries but at least it fixed cases like in ActiveAdmin. Default limit can be changed in the settings using this ```f.default_limit = 25```

**Comment by indirect** [comment]
This should be resolved by #201.
